util/pkg/vfs/s3fs.go

```diff
@@ -151,6 +151,14 @@ func (p *S3Path) WriteFile(data io.ReadSeeker, aclObj ACL) error {
 		request.ACL = s3Acl.RequestACL
 	}
 
```

Comment:
I'm wondering whether this will work with non-S3 filesystems (eg. Digital Ocean's VFS). 

I think we should leave request.SSEKMSKeyId and request.ServerSideEncryption both unset if p.sse is false and probably raise a warning if it's false and the KOPS_STATE_S3_SSEKMSKEY is set.

@andrewsykim, what are your thoughts?

Reply:
Yes, please do a validation against p.sse :)